db/seeds.rb

```diff
@@ -30,10 +30,12 @@
 now = Time.zone.today
 User.all.each do |user|
   5.times do |i|
+    bef_pick = rng.rand(2)
     Score.create!(
       user: user,
-      total_score: rng.rand(66..99),
-      played_at: now - 5.days + i.days
+      total_score: bef_pick.zero? ? rng.rand(27..90) : rng.rand(90..180),
+      played_at: now - 5.days + i.days,
+      number_of_scores: bef_pick.zero? ? 9 : 18
```

Comment:
I know it's correct, but it's a bit confusing.
1. Cannot figure out where the variable name `bef_pick` comes from.
2. We take a random integer between 0 and 1
- if it's 0, apply 9-hole case
- otherwise, apply 18-hole case

Instead of 0 and 1, why don't we use specifically 9 and 18? Take a random choice between 9 and 18 and apply cases.